indico/modules/rb/operations/bookings.py

```diff
@@ -425,14 +425,21 @@ def split_booking(booking, new_booking_data):
             'recurrence_weekdays': booking.recurrence_weekdays,
         }
 
-        booking.modify(old_booking_data, session.user)
+        # Set extra_fields to None to avoid modifying the old bookings
+        booking.modify(old_booking_data, session.user, extra_fields=None)
 
     for occurrence_to_cancel in occurrences_to_cancel:
         occurrence_to_cancel.cancel(session.user, silent=True)
 
     prebook = not room.can_book(session.user, allow_admin=False) and room.can_prebook(session.user, allow_admin=False)
-    resv = Reservation.create_from_data(room, dict(new_booking_data, start_dt=new_start_dt), session.user,
-                                        prebook=prebook, ignore_admin=True, force_internal_note=True)
+    resv = Reservation.create_from_data(
+        room,
+        dict(new_booking_data, start_dt=new_start_dt, extra_fields=extra_fields),
```

Comment:
We can use this nicer (IMHO) way of extending a dict since we're use a modern Python version nowadays ;)

```suggestion
        new_booking_data | {'start_dt': new_start_dt, 'extra_fields': extra_fields},
```